src/icons/share2.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
1. It's much better if the name illustrates the content of the icon. I think the `share-dots.svg` or `connections.svg` are more suitable;
2. Incorrect icon was imported. The one in figma has `width=14` and `height=16`